q2_fmt/tests/test_engraftment.py

```diff
@@ -791,12 +791,13 @@ def test_get_donor(self):
                                         'transfered_donor_features',
                                         'total_donor_features', 'donor',
                                         'subject', 'group'])
-        peds_df = _compute_peds(peds_df=peds_df, peds_type="Sample",
-                                peds_time=np.nan,
-                                reference_series=reference_series,
-                                table=table_df, metadata=metadata_df,
-                                time_column="group", reference_column="Ref",
-                                subject_column="subject")
+        peds_df = _compute_proportion(df=peds_df, type="Sample",
```

Comment:
```suggestion
        pedf_df = _compute_proportion(df=pedf_df, type="Sample",
```
also wondering if you're keeping 'sample' as the type? i know the others were updated to the abbreviation, so not sure if you wanted to be consistent there (since this is just internal - not that users will see it)